.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
+  - name: "💬 Chat"
+    url: https://matrix.to/#/#jenkins-release:libera.chat
+    about: Please ask and answer questions here.
   - name: "❓ IRC"
     url: https://web.libera.chat/
-    about: Please ask and answer questions here, (#jenkins-release on libera.chat).
+    about: Click here to access IRC from a browser (#jenkins-release on libera.chat).
```

Comment:
click here doesn't make sense as this text isn't hyperlinked

Reply:
How about `If you don't use Matrix, connect via IRC on libera.chat` ?

Reply:
could do although I think we could just remove it, most people who aren't already in are likely to prefer matrix.